components/Elevation/tests/unit/MDCMaterialElevationRespondingTests.m

```diff
@@ -535,4 +535,19 @@ - (void)testElevationDidChangeForSuperviewBeforeSubviewsBlockIsCalled {
   XCTAssertEqualWithAccuracy(viewBaseElevation, fakeOverrideElevation, 0.001);
 }
 
+- (void)testAbsoluteElevationReturnsCorrectValue {
+  // Given
+  CGFloat fakeElevation = 3;
+  CGFloat fakeCurrentElevation = 20;
+  self.elevationOverrideViewController.elevation = fakeCurrentElevation;
+  self.elevationOverrideViewController.mdc_overrideBaseElevation = fakeElevation;
+
+  // When
+  [self.elevationOverrideViewController.view addSubview:self.view];
+
+  // Then
+  XCTAssertEqualWithAccuracy(self.view.mdc_baseElevation, self.view.mdc_absoluteElevation,
+                             0.001);
+}
+
```

Comment:
Please add tests for the varied containment/override arrangements as done for the other properties.

For example:

1.  ElevatableView -> OverrideView
2. OverrideView -> ElevatableView
3. ElevatableView -> View -> OverrideView
4. OverrideView -> View -> ElevatableView
5. ElevatableView -> OverrideView -> View

Reply:
done